src/ui/layout/qgslayoutdesignerbase.ui

```diff
@@ -1085,42 +1086,50 @@
     <string>Normal</string>
    </property>
   </action>
-  <action name="mActionPreviewModeGrayscale">
+  <action name="mActionPreviewModeMono">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
-    <string>Simulate Photocopy (&amp;Grayscale)</string>
+    <string>Simulate Fax (&amp;Mono)</string>
```

Comment:
Wonder if we can kill this "fax" annotation here and just go with "Simulate Monochrome"?

Reply:
Agreed, modified by 989cd04.